static/app/components/smartSearchBar/searchDropdown.tsx

```diff
@@ -95,7 +95,12 @@ const SearchDropdown = ({
             </Button>
           ))}
       </ButtonBar>
-      <Button size="xs" href="https://docs.sentry.io/product/sentry-basics/search/">
+      <Button
+        size="xs"
+        href="https://docs.sentry.io/product/sentry-basics/search/"
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
use `external` instead of these two props

Reply:
I think there is a `openInNewTab` prop too

Reply:
https://github.com/getsentry/sentry/blob/b2839cf8352c17913298c69ce640497b192a11b2/static/app/components/links/externalLink.tsx#L8 actually we may have to add that prop into `Button`

Reply:
it seems like `externalLink` [defaults `openInNewTab` as true](https://github.com/getsentry/sentry/blob/4ebf875314334710ba2696d42c7d5a0bc203bfe0/static/app/components/links/externalLink.tsx#L12), so we don't need to add that prop to open this button in a new window.